### PR TITLE
Remove warnings from 01_dirty_categories.py

### LINKED ISSUE #704: Remove warnings from 01_dirty_categories.py
### Describe the issue linked to the documentation

1. In the first notebook (Example 1) i get warnings for one-hot encoder:
```
/home/circleci/project/miniconda/envs/testenv/lib/python3.10/site-packages/sklearn/preprocessing/_encoders.py:972: FutureWarning: `sparse` was renamed to `sparse_output` in version 1.2 and will be removed in 1.4. `sparse_output` is ignored unless you leave `sparse` to its default value.
```

and additionally for the 'minhash' encoder:
```
/home/circleci/project/miniconda/envs/testenv/lib/python3.10/site-packages/sklearn/base.py:464: UserWarning: X does not have valid feature names, but MinHashEncoder was fitted with feature names
```

image

2. Why do we need this in describing 'Employee Position Title' column:
```python
import numpy as np
np.unique(y)
```

### Suggest a potential alternative/fix

_No response_

### PR DESCRIPTION
**Pull request by kazmierczakP**
Fix #704 
The second warning is still being investigated.

### CODE REVIEW

File: examples/01_dirty_categories.py [hunk 1]
```diff
@@ -261,6 +259,7 @@
 # Let's perform a cross-validation to see how well this model predicts:
 
 from sklearn.model_selection import cross_val_score
+import numpy as np
```
Comment (jovan-stojanovic) [inline examples/01_dirty_categories.py:262]: Hi, thanks for this PR! 
Here, I believe that you need to import numpy earlier (for line 182).

Comment (kazmierczakP) [reply examples/01_dirty_categories.py:262]: Right, I see 
Should I close pull request and create new one or somehow edit this one?

Comment (jovan-stojanovic) [reply examples/01_dirty_categories.py:262]: No, you can edit this one and iterate as many times as you want!
You can do this by making [new commits and then pushing them](https://www.earthdatascience.org/workshops/intro-version-control-git/basic-git-commands/) to github.

Comment (Vincent-Maladiere) [reply examples/01_dirty_categories.py:262]: Hi @kazmierczakP, you only have to `git commit ...` and `git push` on your laptop, and then the commits will land on this PR :)

**Comment by kazmierczakP** [comment]
```
All changes made by hooks:
diff --git a/examples/01_dirty_categories.py b/examples/01_dirty_categories.py
index b3ec700..798e4e1 100644
--- a/examples/01_dirty_categories.py
+++ b/examples/01_dirty_categories.py
@@ -141,7 +141,7 @@ pipeline.fit(X, y)
 #
 # The |OneHotEncoder| is actually not well suited to the 'Employee
 # Position Title' column, as this column contains 400 different entries:
-X['employee_position_title'].value_counts()
+X["employee_position_title"].value_counts()
```
^^ Is it just the formating (' ' instead of " ")?

**Comment by Vincent-Maladiere** [comment]
Yes, you need to activate the pre-commit that will run these formatting operations for you:
```shell
pip install pre-commit
pre-commit init
pre-commit run
```
This will be run automatically at each commit.

**Review by Vincent-Maladiere — APPROVED** [review]
Looks good, thanks @kazmierczakP :)

**Pull request merged**